docs/extensions/guides/renderer-extension.md

```diff
@@ -200,24 +214,13 @@ export default class ExampleExtension extends LensRendererExtension {
 }
 ```
 
-The above defines two cluster pages and three cluster page menu objects.
-The cluster page definitons are straightforward.
-The first cluster page menu object defines the parent of a foldout submenu.
-Setting the `id` field in a cluster page menu definition implies that it is defining a foldout submenu.
-Also note that the `target` field is not specified (it is ignored if the `id` field is specified).
-This cluster page menu object specifies the `title` and `components` fields, which are used in displaying the menu item in the cluster dashboard sidebar.
-Initially the submenu is hidden.
-Activating this menu item toggles on and off the appearance of the submenu below it.
-The remaining two cluster page menu objects define the contents of the submenu.
-A cluster page menu object is defined to be a submenu item by setting the `parentId` field to the id of the parent of a foldout submenu, `"example"` in this case
+The above defines two cluster pages and three cluster page menu objects. The three cluster page menu objects include one parent menu item and two sub menu items. Parent items require an `id` value, whereas sub items require a `parentId` value. The value of the sub item `parentId` will match the value of the corresponding parent item `id`. Parent items don't require a `target` value. Assign values to the remaining properties as explained above. 
 
 ### `globalPages`
 
-Global pages appear independently of the cluster dashboard and they fill the Lens UI space.
-A global page is typically triggered from the cluster menu using a [global page menu](#globalpagemenus).
-They can also be triggered by a [custom app menu selection](../main-extension#appmenus) from a Main Extension or a [custom status bar item](#statusbaritems).
-Global pages can appear even when there is no active cluster, unlike cluster pages.
-It is conventional to use a global page to show information and provide functionality relevant across clusters, along with custom data and functionality that your extension may have.
+Global pages are independent of the cluster dashboard and they fill the entire Lens UI. Their primary use is to display information and provide functionality across clusters, including customized data and functionality unique to your extension. 
```

Comment:
Let's say "...they can fill the entire... " because I've learned that it depends on how you define your page layout.